src/modules/hunting/capabilities.py

```diff
@@ -0,0 +1,41 @@
+import socket
+import logging
+
+from ..discovery.hosts import RunningAsPodEvent
+
+from ...core.events import handler
+from ...core.events.types import Event, Vulnerability
+from ...core.types import Hunter, AccessRisk, KubernetesCluster
+
+
+class CapNetRawEnabled(Event, Vulnerability):
+    """CAP_NET_RAW is enabled by default for pods, an attacker could potentially perform network attacks on other pods running on the same node"""
```

Comment:
Better to clarify that this is only a problem if an attacker compromises a pod. How about "CAP_NET_RAW is enabled by default for pods. If an attacker compromises a pod, they can potentially use this capability to perform network attacks on other pods running on the same node"